src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const objValueOfDate = {};
+  const arrDateData = date.split(fromFormat[3]);
+  const newArrDate = [];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    objValueOfDate[fromFormat[i]] = arrDateData[i];
+  }
+
+  if (toFormat.includes('YYYY') && !('YYYY' in objValueOfDate)) {
+    objValueOfDate['YYYY'] = objValueOfDate['YY'] < 30
+      ? 2000 + Number(objValueOfDate['YY'])
```

Comment:
You are using the Number constructor to convert a string into a number. Consider using the unary plus operator (+) for this conversion as it is more concise and faster.